charts/dapr/charts/dapr_operator/values.yaml

```diff
@@ -50,3 +50,5 @@ debug:
   enabled: false
   port: 40000
   initialDelaySeconds: 30000
+
+allowedServiceResources: "CronJob,Job,DaemonSet"
```

Comment:
Please, also add other types, like Deployment and StatefulSet

Reply:
> Please, also add other types, like Deployment and StatefulSet

The logic for adding servies to Deployment and StatefulSet existed before.https://github.com/dapr/dapr/blob/master/pkg/operator/handlers/dapr_handler.go#L106

Are you referring to tombstoning the original ones and adding them here to achieve a unified way of adding services?

Reply:
cc @artursouza